src/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.cpp

```diff
@@ -294,15 +296,15 @@ namespace NovelRT::ResourceManagement::Desktop
     {
         filePath = _resourcesRootDirectory / filePath;
 
-        jsoncons::json j(jsoncons::json_object_arg);
-        std::vector<jsoncons::json> memberMetadataJson{};
+        nlohmann::json j{};
+        nlohmann::json memberMetadataJson{};
```

Comment:
Is there a reason this isn't a std::vector of the json object anymore?

Reply:
The json object behaves like std::vector, i feel like having it be a json object instead would make more sense in showing what it actually is.

Reply:
To confirm, do we have a test covering this code path in the case the vector would have multiple entries?